docs/class-definitions.md

```diff
@@ -279,25 +333,6 @@ Set your current location. [Location](#location-1) can be any JSON-serializable
 ```ts
 type set = (update: Location) => void;
 ```
-### getSelf
-Get location for self
-```ts
-await space.locations.getSelf()
-```
-### getAll
-Get location for all members
-
-```ts
-await space.locations.getAll()
-```
-### getOthers
-Get location for other members
-
-```ts
-await space.locations.getOthers()
-```
-
-
 
 ### subscribe
```

Comment:
Can we move subscribe before set please in terms of the order of the methods? Thanks